frappe/core/doctype/file/file.py

```diff
@@ -1052,4 +1052,4 @@ def attach_files_to_document(doc, event):
 				folder="Home/Attachments",
 			).insert()
 		except Exception:
-			frappe.log_error(title=_("Error Attaching File"))
+			file_doc.log_error("Error Attaching File")
```

Comment:
Handling this exception leads to a `NameError` since the variable `file_doc` won't be defined if the insert isn't successful.

Will fix this via #16289

Reply:
Thanks @gavindsouza !